easybuild/framework/easyconfig/tools.py

```diff
@@ -472,7 +472,7 @@ def find_related_easyconfigs(path, ec):
     if len(parsed_version) >= 2:
         version_patterns.append(r'%s\.%s\.\w+' % tuple(parsed_version[:2]))  # major/minor version match
     if parsed_version != parsed_version[0]:
-        version_patterns.append(r'%s\.[\d-]+\.\w+' % parsed_version[0])  # major version match
+        version_patterns.append(r'%s\.[\d-]+(\.\w+)*' % parsed_version[0])  # major version match
```

Comment:
why not just like this:
```suggestion
        version_patterns.append(r'%s\.\w+' % parsed_version[0])  # major version match
```

Reply:
@smoors this makes `test_find_related_easyconfigs` fail, at least as it is now, because it also matches easyconfigs with different toolchains and versionsuffixes (note that these `version_patterns` are only part of the actual regex, the toolchain and versionsuffixes patterns are inserted in the next block)

to be clear, although my suggestion doesn't make the tests fail, I'm also not sure if it won't change the behaviour in some unintended way, which is way I marked the PR as draft :)